owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/users/GetRemoteUserInfoOperation.kt

```diff
@@ -29,7 +29,8 @@ import com.owncloud.android.lib.common.http.methods.nonwebdav.GetMethod
 import com.owncloud.android.lib.common.operations.RemoteOperation
 import com.owncloud.android.lib.common.operations.RemoteOperationResult
 import com.owncloud.android.lib.common.operations.RemoteOperationResult.ResultCode
-import com.owncloud.android.lib.resources.CommonOcsResponse
+import com.owncloud.android.lib.resources.response.CommonOcsResponse
```

Comment:
Do you add these two imports but not use them in this commit? How did you get UserInfoResponse before adding its import?

Reply:
I just moved CommonOcsResponse to another folder. I don't understand your question